challenges/objects-arrays.js

```diff
@@ -15,20 +35,28 @@
 // Using your dinosaur objects, log answers to these questions:
 
 // How much did tyrannosaurus weigh?
-console.log();
+weight(){
+  console.log(`The ${tyrannosaurus.name} weighs ${tyrannosaurus.weight}`);
+}
 
 // What was the diet of a velociraptor?
-console.log();
+diet(){ 
+  console.log(`The ${velociraptor.name} has a ${stegosaurus.diet}`);
+  
+}
 
 // How long was a stegosaurus?
-console.log();
+length(){
```

Comment:
You can leave out the creation of new methods here! And below, console.log will work fine alone!